src/constructs/ecsIsoServiceAutoscaler/ecsIsoServiceAutoscaler.ts

```diff
@@ -127,7 +159,31 @@ export class EcsIsoServiceAutoscaler extends Construct {
       description: `Kicks off Lambda to adjust ECS scaling for service: ${props.ecsService.serviceName}`,
       enabled: true,
       schedule: Schedule.rate(Duration.minutes(1)),
-      targets: [new LambdaFunction(ecsScalingManager)],
+      targets: [new LambdaFunction(this.ecsScalingManagerFunction)],
     });
+
+    if (!props.role) {
```

Comment:
Would also be nice if the role was exposed as a property.

E.g.,

```
this.role = props.role || new iam.Role(this, 'ServiceRole', {
  assumedBy: new iam.ServicePrincipal('lambda.amazonaws.com'),
  managedPolicies,
});
```

Reply:
Part of the change in this PR includes [exposing](https://github.com/cdklabs/cdk-enterprise-iac/pull/43/files#diff-6a3b40df0eff41157e397f3ca8f7d63a4450e11870a8928941b997afeb0bf6eeR131) the lambda function itself, so the role should already be available in `this.ecsScalingManagerFunction.role`